app/views/dashboard/arms/create.js.coffee

```diff
@@ -23,7 +23,7 @@ $("#modal_errors").html("<%= escape_javascript(render(partial: 'shared/modal_err
 $("#per_patient_services").html("<%= escape_javascript(render(:partial =>'dashboard/sub_service_requests/per_patient_per_visit', locals: {sub_service_request: @sub_service_request, service_request: @service_request})) %>");
 
 $("#sub_service_request_header").html("<%= escape_javascript(render(partial: 'dashboard/sub_service_requests/header', locals: { sub_service_request: @sub_service_request })) %>");
-$("#subsidy_information").html("<%= escape_javascript(render(partial: 'dashboard/subsidies/subsidy', locals: { sub_service_request: @sub_service_request })) %>");
+$("#subsidy_information").html("<%= escape_javascript(render(partial: 'dashboard/subsidies/subsidy', locals: { sub_service_request: @sub_service_request, admin: true })) %>");
```

Comment:
While I agree with Andrew that this fix is out of scope for this particular pull request, it is a necessary change that I'm sure without he was not able to access all the necessary parts of the app to accomplish this task. Either way, this fix needs to come into the bootstrap branch.